quilt/tools/command.py

```diff
@@ -62,6 +62,12 @@ def _save_auth(auth):
 def _handle_response(resp, **kwargs):
     if resp.status_code == requests.codes.unauthorized:
         raise CommandException("Authentication failed. Run `quilt login` again.")
+    elif not resp.ok:
```

Comment:
Are we now exclusively using 200 as the response code for successful operations (i.e. even for deletes, etc. that sometime return other values)?